admin/database/database.go

```diff
@@ -207,17 +209,19 @@ type Project struct {
 	Description          string
 	Public               bool
 	Region               string
-	GithubURL            *string   `db:"github_url"`
-	GithubInstallationID *int64    `db:"github_installation_id"`
-	Subpath              string    `db:"subpath"`
-	ProdBranch           string    `db:"prod_branch"`
-	ProdVariables        Variables `db:"prod_variables"`
-	ProdOLAPDriver       string    `db:"prod_olap_driver"`
-	ProdOLAPDSN          string    `db:"prod_olap_dsn"`
-	ProdSlots            int       `db:"prod_slots"`
-	ProdDeploymentID     *string   `db:"prod_deployment_id"`
-	CreatedOn            time.Time `db:"created_on"`
-	UpdatedOn            time.Time `db:"updated_on"`
+	GithubURL            *string       `db:"github_url"`
+	GithubInstallationID *int64        `db:"github_installation_id"`
+	Subpath              string        `db:"subpath"`
+	ProdBranch           string        `db:"prod_branch"`
+	ProdVariables        Variables     `db:"prod_variables"`
+	ProdOLAPDriver       string        `db:"prod_olap_driver"`
+	ProdOLAPDSN          string        `db:"prod_olap_dsn"`
+	ProdSlots            int           `db:"prod_slots"`
+	ProdDeploymentID     *string       `db:"prod_deployment_id"`
+	ProductionTTL        time.Duration `db:"production_ttl"`
+	PreviewTTL           time.Duration `db:"preview_ttl"`
```

Comment:
Make it `ProdTTL` instead to align with other fields. And we can remove `PreviewTTL` until we implement previews (didn't add it for other fields either yet)